homeassistant/components/sensor/darksky.py

```diff
@@ -96,6 +96,9 @@
     'precip_intensity_max': ['Daily Max Precip Intensity',
                              'mm', 'in', 'mm', 'mm', 'mm', 'mdi:thermometer',
                              ['currently', 'hourly', 'daily']],
+    'uv_index': ['UV Index', None, None, None, None, None,
```

Comment:
UV Index is a unitless measurement, but without a unit, it won't show up well in history graphs.

Is there a standard in HASS for unitless numeric measurement units?

Reply:
I'd say that although there generally isn't a formal unit attached, UVI isn't a ratio/fraction/count or something else truly unit-less. It just is it's own unit. Hard to find some precedence, but in the [Wikipedia Description](https://en.wikipedia.org/wiki/Ultraviolet_index#Description), they do use the phrase +- 1 UV Index Unit.

We've got one precedent I could find searching the repo here: https://github.com/home-assistant/home-assistant/blob/46e030662da4ab7c653a23e1f34e527c9a0e7a8f/homeassistant/components/sensor/isy994.py#L80

If you could add that string to [const.py](https://github.com/home-assistant/home-assistant/blob/4cf300a7100edd61b7c558d3ef1cc33b5e1fc0cb/homeassistant/const.py#L250-L280), as `UV_INDEX` or `UNIT_UV_INDEX`, you can then import and use it here as the unit, and replace the usage in isy994. It's not perfect, but that will at least give us a formal precedent for how we handle it.